neqo-common/src/hrtime.rs

```diff
@@ -301,7 +301,9 @@ impl Time {
         #[cfg(windows)]
         {
             if let Some(p) = self.active {
-                assert_eq!(0, unsafe { timeBeginPeriod(p.as_uint()) });
+                unsafe {
+                    timeBeginPeriod(p.as_uint());
+                };
```

Comment:
Are we likely to get an unused return value warning?  Should this be:

```suggestion
                _ = unsafe { timeBeginPeriod(p.as_uint()) };
```

Reply:
Will do.
Thanks.